WordPressKit/HTTPProtocolHelpers.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+extension String.Encoding {
+    init?(ianaCharsetName: String) {
```

Comment:
TIL about IANA, the Internet Assigned Numbers Authority. What do you think of adding a link?


```suggestion
    /// See: https://www.iana.org/assignments/character-sets/character-sets.xhtml
    init?(ianaCharsetName: String) {
```